app/views/submissions/repo_browser.html.erb

```diff
@@ -97,6 +97,8 @@
                    id: @grouping.id,
                    assignment_id: @assignment.id) do %>
       <%= hidden_field_tag 'current_revision_number', @revision_number %>
+      <label><%= check_box_tag(:apply_late_penalty) %><%=t('apply_late_penalty')%></label>
```

Comment:
You want to use one `check_box_tag` and then one `label_tag` for the label. See http://apidock.com/rails/v4.2.1/ActionView/Helpers/FormTagHelper/label_tag